docs/user/vsphere/privileges.md

```diff
@@ -0,0 +1,50 @@
+# Required Privileges
+In order to install an OpenShift cluster to a vCenter, the user provided to the installer needs privileges to read and create the necessary resources. The easiest way to achieve this level of permission and ensure success is to install with a user who has administrative privileges.
+
+If the provided user has admin privileges, no action is required and you can skip to the next step. Otherwise, the rest of this document can be used as a resource to create a user with more fine-grained privileges.
+
+## Privileges
+In order to create an OpenShift cluster, a user needs permissions for the following categories: _Datastore_, _Folder_, _Host_, _vSphere Tagging_, _Network_, _Resource_, _Profile-driven storage_, _vApp_, and _Virtual machine_.
```

Comment:
That same user will be assigned as the cloud provider account as well. We should probably align the required permission to the dynamic provisioning docs: https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/vcp-roles.html